dev-tools/es_release_notes.pl

```diff
@@ -32,7 +32,7 @@
     ">enhancement", ">bug",           ">regression",  ">upgrade"
 );
 my %Ignore = map { $_ => 1 }
-    ( ">non-issue", ">refactoring", ">docs", ">test", ">test-failure", ":Core/Build" );
+    ( ">non-issue", ">refactoring", ">docs", ">test", ":Core/Build", ">test-failure" );
```

Comment:
is the ordering change necessary?

Reply:
No, I based changes off of what was in 6.3, which was missing the ">test-failure" label, and so I added it there.  When I changed to master I pulled the wrong change.  Fixed